app/code/Magento/Eav/Model/Entity/Attribute/Source/AbstractSource.php

```diff
@@ -31,6 +31,22 @@
      */
     protected $_options = null;
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getAllOptions()
+    {
+        if (empty($this->_options)) {
```

Comment:
I'd suggest using the default `_options` value that is `null` in the condition to avoid multiple calls to `loadOptions` if no options are available.
```suggestion
        if (null === $this->_options) {
```